RevenueCatUI/CustomerCenter/ViewModels/CustomerCenterViewModel.swift

```diff
@@ -146,11 +146,6 @@ import RevenueCat
             return .purchasesNotFound
         }
     }
-
-    func onAppUpdateClick() {
-        // swiftlint:disable:next todo
-        // TODO: implement opening the App Store
-    }
```

Comment:
Since this is done using a ViewController, I decided that it's best for the view layer to handle this.